src/Backupper/AbstractBackupper.php

```diff
@@ -43,9 +46,10 @@ public function checkBinary(): string
     {
         $process = new Process([$this->binary, '--version']);
         $process->run();
+
         if (!$process->isSuccessful()) {
-            throw new \InvalidArgumentException(\sprintf(
-                "Error while trying to process '%s', check configuration for binary '%s",
+            throw new \LogicException(\sprintf(
+                "Error while running \"%s\" command, check configuration for binary '%s'.",
```

Comment:
That's kind of schizophrenic, either enclose with `"` and use `'` or the other way around, but mixing the two is weird.

Reply:
I changed this because the command will probably contain a lot `'` on Linux. I can also change it for the second placeholder if you prefer.